packages/core/modules-sdk/src/remote-link.ts

```diff
@@ -453,4 +455,72 @@ export class RemoteLink {
   ): Promise<[CascadeError[] | null, RestoredIds]> {
     return await this.executeCascade(removedServices, "restore")
   }
+
+  async list(
+    link: LinkDefinition | LinkDefinition[],
+    options?: { asLinkDefinition?: boolean }
+  ): Promise<(object | LinkDefinition)[]> {
+    const allLinks = Array.isArray(link) ? link : [link]
+    // TODO: create a generic function that builds a serviceLinkMap based on data
+    const serviceLinks = new Map<string, object[]>()
+
+    for (const rel of allLinks) {
+      const mods = Object.keys(rel).filter((mod) => mod !== "data")
+
+      if (mods.length > 2) {
+        throw new Error(`Only two modules can be linked.`)
+      }
+
+      const [moduleA, moduleB] = mods
+      const pk = Object.keys(rel[moduleA])
+      const moduleAKey = pk.join(",")
+      const moduleBKey = Object.keys(rel[moduleB]).join(",")
+
+      const service = this.getLinkModule(
+        moduleA,
+        moduleAKey,
+        moduleB,
+        moduleBKey
+      )
+
+      if (!service) {
+        throw new Error(
+          linkingErrorMessage({
+            moduleA,
+            moduleAKey,
+            moduleB,
+            moduleBKey,
+            type: "link",
+          })
+        )
```

Comment:
do you mind to create a private function `getLinkModuleOrThrow`? it can also be public.
there are now 3 places doing that same thing.

Reply:
Yes sure. There's a TODO on line 464 to do this exactly. I wanted to get your ✅ on the approach before I do that